backend/src/main/resources/application-azure-dev.yaml

```diff
@@ -3,6 +3,7 @@ spring:
 simple-report:
   azure-reporting-queue:
     enabled: true
+    callback-enabled: true
```

Comment:
I wonder if `callback` is the best term for this concept, since its an overloaded term. should we rename it to `exception-callback-enabled` or `report-exceptions-enabled`

Reply:
What about `exception-webhook-enabled`?

Reply:
sounds good to me!

Reply:
Updated in 0be9fa2 – please sanity check!